src/vm/gccover.cpp

```diff
@@ -1692,22 +1692,38 @@ void DoGcStress (PCONTEXT regs, MethodDesc *pMD)
 
 #if defined(_TARGET_X86_) || defined(_TARGET_AMD64_) || defined(_TARGET_ARM_) || defined(_TARGET_ARM64_)
     FrameWithCookie<GCFrame> gcFrame;
-    DWORD_PTR retVal = 0;
+
+    OBJECTREF* retPointer = 0;
+    UINT numObjRefs = 1;
+#if defined(UNIX_AMD64_ABI) || defined(_TARGET_ARM64_)
+    // Multireg return.
```

Comment:
Just to be clear, I might say "These targets support multireg returns."

Reply:
Fixed, thank you.